### PR TITLE
add me to watching things; alphebetize lists

### PR DESCRIPTION
**Pull request by edunham**
I would like to watch some things. 

I also went `:sort` on the lists to which I added myself, so that future additions can be done with a less noisy diff.

### CODE REVIEW

**Comment by jdm** [comment]
Bah, https://github.com/servo/highfive/blob/master/handlers/label_watchers/tests/dont_annoy.json was written without using test-only data so it no longer passes with these additional changes.

File: handlers/watchers/watchers.ini [hunk 1]
```diff
@@ -1,12 +1,13 @@
 [servo/servo]
-emilio = components/script/dom/webgl components/canvas/webgl ports/geckolib components/style/ components/layout/
-bholley = components/style/
-Wafflespeanut = python/
-jgraham = components/webdriver_server/ tests/wpt/ -tests/wpt/mozilla -tests/wpt/metadata -tests/wpt/web-platform-tests -tests/wpt/metadata-css -tests/wpt/css-tests
 KiChjang = components/script components/script_traits components/net components/net_traits
+Wafflespeanut = python/
```
Comment (KiChjang) [inline handlers/watchers/watchers.ini]: Might as well change this back to a lowercase w.

Comment (wafflespeanut) [inline handlers/watchers/watchers.ini]: Shouldn't make a difference technically (Github usernames and labels are case-insensitive, and highfive too), but for consistency, you bet :)

**Comment by edunham** [comment]
Testing works in mysterious ways. I poked the test to make it not fail with the current watcher lists. I'm not totally sure if that was the *right* fix, and it seems like it's almost certainly not the *best* fix, but it does appear to be *a* fix for the problem at hand.

**Comment by canaltinova** [comment]
FYI, I filed a PR to use test data instead of real repo data. You can revert that test change after that probably.

**Comment by jdm** [comment]
Rebased the important parts in #185. Sorry about the delay.

**Comment by edunham** [comment]
Thanks!

**Pull request closed without merging**